### ISSUE TITLE
False positive: systemd-service-without-service_del_postun if service restart is disabled

### ISSUE THREAD

**Issue by Vogtinator** [openSUSE/rpmlint-checks#72]
The sddm service must not be restarted on upgrades, so the `%postun` scriptlet contains:

```
%postun
# Don't restart on upgrades (boo#1161826)
%if 0%{?suse_version} > 1500
%service_del_postun_without_restart sddm.service
%else
%service_del_postun -n sddm.service
%endif
```

This leads to:

```
[  129s] sddm.x86_64: E: systemd-service-without-service_del_postun sddm.service
[  129s] The package contains a systemd service but doesn't contain a %postun with a
[  129s] call to service_del_postun.
```

The expanded scriptlet does not mention the unit names at all:

```
postuninstall scriptlet (using /bin/sh):
# Don't restart on upgrades (boo#1161826)

if [ -x /usr/bin/systemctl ]; then
        /usr/bin/systemctl daemon-reload || :
fi
```
